tul_cob_pipeline_dag.py

```diff
@@ -15,8 +15,27 @@
 from cob_datapipeline.processtrajectlog import process_trajectlog
 from cob_datapipeline.task_solrgetnumdocs import task_solrgetnumdocs
 
-core_name = Variable.get("BLACKLIGHT_CORE_NAME")
+#
+# INIT SYSTEMWIDE VARIABLES
+#
+# check for existence of systemwide variables shared across tasks that can be
+# initialized here if not found (i.e. if this is a new installation)
+#
+try:
+    oai_publish_interval = Variable.get("ALMA_OAI_PUBLISH_INTERVAL")
+    dag_run_interval = timedelta(hours=int(oai_publish_interval))
+except KeyError:
+    Variable.set("ALMA_OAI_PUBLISH_INTERVAL", "6")
+    dag_run_interval = timedelta(hours=6)
+
+try:
+    date_last_harvest = Variable.get("almaoai_last_harvest_date")
+except KeyError:
+    Variable.set("almaoai_last_harvest_date", datetime.datetime.now().strftime('%Y-%m-%dT%H:%M:%SZ'))
```

Comment:
and setting a default of now as the last_harvest_date sets us up to then remove 6 hours (or the interval) from it by default, so we get the last interval by default - if I'm reading this correctly?